x-pack/plugins/canvas/server/routes/workpad.js

```diff
@@ -97,6 +97,25 @@ export function workpad(server) {
 
       return savedObjectsClient
         .get(CANVAS_TYPE, id)
+        .then(obj => {
```

Comment:
I think you were intending to fix the problem by modifying the object that comes back from the API, but ultimately it seems to have just been an oversight in `StaticWorkpadPage`.

I don't think you need this code, you don't seem the fix works the same way without it. That said, are the group objects in the element list supposed to be there? If not, do we need to open another issue about that problem?

Reply:
I'll open an issue to investigate, there's no harm if some groups end up on the element list, esp. that this change will re-persist them correctly on the first update of any kind; there's no technical difference between the containers (merely convention); and I should check which releases were impacted in this timeframe

Reply:
#37562 opened